.travis.yml

```diff
@@ -11,57 +11,33 @@ cache:
   directories:
   - zookeeper
 language: python
-python:
-- '2.7'
-matrix:
-  exclude:
-  - python: '2.7'
+jobs:
   include:
-  - python: '2.7'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py27
-  - python: '2.7'
-    env: ZOOKEEPER_VERSION=3.5.7 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py27
-  - python: '2.7'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py27-gevent
-  - python: '2.7'
-    env: ZOOKEEPER_VERSION=3.5.7 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py27-gevent
-  - python: '2.7'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py27-eventlet
-  - python: '2.7'
-    env: ZOOKEEPER_VERSION=3.5.7 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py27-eventlet
   - python: '3.7'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py37
+    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py37-gevent-sasl
   - python: '3.7'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py37-sasl
-  - python: '3.7'
-    env: ZOOKEEPER_VERSION=3.5.7 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py37
-  - python: '3.7'
-    env: ZOOKEEPER_VERSION=3.5.7 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py37-sasl
-  - python: '3.8'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py38
-  - python: '3.8'
-    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py38-sasl
+    env: ZOOKEEPER_VERSION=3.5.8 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py37-gevent-sasl
   - python: '3.8'
-    env: ZOOKEEPER_VERSION=3.5.7 ZOOKEEPER_PREFIX="apache-" ZOOKEEPER_SUFFIX="-bin" ZOOKEEPER_LIB="lib" TOX_VENV=py38 DEPLOY=true
+    env: ZOOKEEPER_VERSION=3.4.14 TOX_VENV=py38-gevent
```

Comment:
Should SASL be tested also with CPython 3.8?

Reply:
fixed, thanks!